spring-security/README.md

```diff
@@ -1,3 +1,8 @@
+### :warning: Deprecation Notice
+In alignment with the end of support for [Spring Boot 2.x](https://spring.io/projects/spring-boot#support) and [Spring Framework 5.x](https://spring.io/projects/spring-framework#support), we will also discontinue support for this library by the **end of 2025** and for the corresponding Spring Boot library [spring-security-starter](../spring-security-starter) by **Nov, 2023**.
```

Comment:
Suggestions: "[...] discontinue support for **version 2.x of this library** by [...]"